doc/dnf5.conf.5.rst

```diff
@@ -1141,32 +1147,150 @@ Users can define custom config options in this way.
 ``Distribution Configuration Files``
     /usr/share/dnf5/libdnf.conf.d/50-something.conf
 
-Example configuration:
+Drop-in repo directories
+========================
+
+After the repository configurations are loaded other repo configurations can be overloaded from the directories :ref:`user repos override directory <user_repos_override_dir-label>` and :ref:`distribution repos override directory <distro_repos_override_dir-label>`.
+
+The format of the files inside the directories is the same as the format of the repository configuration files. The options in the overridden files can modify existing repos but cannot create new repositories.
+
+Override files support globs in the repository ID in order to support bulk modifications of repository parameters.
+
+The repository overrides are processed following this order:
+
+1. Files from ``/usr/share/dnf5/repos.override.d/`` and ``/etc/dnf5/repos.override.d/`` are loaded in an alphabetically sorted list. In case files have the same name, the file from ``/usr/share/dnf5/repos.override.d/`` is used. This implies the list has only unique filenames. This also implies that the repository configuration files can be simply masked by creating a file with the same name in the ``/usr`` override directory.
```

Comment:
The order does not match "Example configuration" section. In the example 60-something2.repo is taken from /etc, while this paragraph reads "file from /usr[…] is used".

Reply:
I have a PR https://github.com/rpm-software-management/dnf5/pull/1356 with the config-manager plugin documentation. I also mention there the order of application of repositories configuration overrides. Maybe it will help here.

Reply:
Thanks. I'll take example from that PR. But overall, I think the content is right regarding the implementation. Am I saying wrong stuff? @jrohel

Reply:
> In case files have the same name, the file from ``/usr/share/dnf5/repos.override.d/`` is used.

No. The file from ``/etc/dnf5/repos.override.d/`` is used.

>  can be simply masked by creating a file with the same name in the ``/usr`` override directory.

No. ...in the ``/etc`` override directory.

The idea is that system configuration (in `/etc`, created by the system administrator) can mask distribution's default configuration (in the `/usr`).

Look at the CI test https://github.com/rpm-software-management/ci-dnf-stack/blob/6d80409e967b67d601d73223c0f39ee2e703dd50/dnf-behave-tests/dnf/config-repos-overrides.feature#L35

Reply:
so, I fixed the docs, and I cross-referenced the plugin and this page.


A note on the general status of our docs: there is some duplicate information, which is not necessarily bad but it can be improved.

I would ask, in the future the doc team to review our dnf5 docs to uniform the style and give guidance.

Reply:
For example, we have repetitions in the heading of some sections, or some subsections are not in the same place for plugins/commands. Also, the docs are missing cross-references in general, which brings me to another point that is inconsistency in the label naming. I feel like all the docs should be reviewed once dnf5 lands in rawhide and there is enough time to prioritize it.

Just a general thought, unrelated to this specific PR